src/styles/partials/components/_images.scss

```diff
@@ -0,0 +1,42 @@
+.dg_image {
```

Comment:
this should be nested below `#main-content`

Reply:
done